backend/infrahub/core/schema/definitions/core.py

```diff
@@ -849,7 +850,26 @@
                 },
             ],
             "relationships": [
-                {"name": "tokens", "peer": InfrahubKind.ACCOUNTTOKEN, "optional": True, "cardinality": "many"},
+                {"name": "tokens", "peer": InfrahubKind.ACCOUNTTOKEN, "optional": True, "cardinality": "many"}
+            ],
+        },
+        {
+            "name": "BasePermission",
+            "namespace": "Core",
+            "description": "A permission grants right to a user",
```

Comment:
Minor comment here but a permission grants a right to a role and not a user directly?